src/routes/(protected)/library/+page.svelte

```
@@ -68,21 +85,40 @@
         url.searchParams.set("page", "1");
         $formData.page = 1;
 
-        goto(url.toString(), {
+        await goto(url.toString(), {
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)